lib/actions/sync.coffee

```diff
@@ -79,6 +91,8 @@ module.exports =
 		,
 	]
 	action: (params, options, done) ->
+		fs = require('fs')
```

Comment:
I generally run `resin-lint` to catch unused variables - we use all of the imported modules later on.